proto/cel/expr/conformance/env_config.proto

```diff
@@ -0,0 +1,159 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package cel.expr.conformance;
+
+import "cel/expr/checked.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/descriptor.proto";
+
+option cc_enable_arenas = true;
+option go_package = "cel.dev/expr/conformance";
+option java_multiple_files = true;
+option java_outer_classname = "EnvironmentProto";
+option java_package = "cel.dev.expr.conformance";
+
+// Representation of a CEL Environment, defining what features and extensions
+// are available for conformance testing.
+message Environment {
+  // Name of the environment
+  string name = 1;
+
+  // Description for the current environment
+  string description = 2;
+
+  // Sets the namespace (container) for the expression.
+  // This is used to simplify resolution: e.g. with container
+  // `google.rpc.context`, an identifier of
+  // `google.rpc.context.AttributeContext` could be referred to simply as
+  // `AttributeContext` in the CEL expression.
+  string container = 3;
+
+  // Import represents a type name that will be abbreviated by its simple name
+  // using the cel.Abbrevs() option.
+  message Import {
+    // Qualified type name which will be abbreviated
```

Comment:
Instead of mentioning the cel-go feature, let's add an example of the desired behavior:

An `Import{name: 'google.rpc.Status'}` would ensure that `google.rpc.Status` is available by the simple name `Status` within CEL expressions. Imports are intended to make it easy to reference simple type names from packages other than the expression `container`.